packages/amplify-graphql-conversation-transformer/src/graphql-types/message-model.ts

```diff
@@ -256,3 +283,29 @@ const constructConversationMessageModel = (
 
   return object;
 };
+
+const STREAM_RESPONSE_TYPE_NAME = 'ConversationMessageStreamPart';
+
+export const constructStreamResponseType = (): ObjectTypeDefinitionNode => {
+  return {
+    kind: 'ObjectTypeDefinition',
+    name: { kind: 'Name', value: STREAM_RESPONSE_TYPE_NAME },
+    fields: [
+      makeField('id', [], makeNonNullType(makeNamedType('ID'))),
+      makeField('owner', [], makeNamedType('String')),
+      makeField('conversationId', [], makeNonNullType(makeNamedType('ID'))),
+      makeField('associatedUserMessageId', [], makeNonNullType(makeNamedType('ID'))),
+
+      makeField('contentBlockIndex', [], makeNonNullType(makeNamedType('Int'))),
+
+      makeField('contentBlockText', [], makeNamedType('String')),
+      makeField('contentBlockDeltaIndex', [], makeNamedType('Int')),
+
+      makeField('contentBlockToolUse', [], makeNamedType('AWSJSON')),
+
+      makeField('contentBlockDoneAtIndex', [], makeNamedType('Int')),
+
+      makeField('stopReason', [], makeNamedType('String')),
+    ],
+  };
+};
```

Comment:
This currently isn't being used, the type comes from data-schema.
However, that is changing in a follow up PR where the supporting types are consolidated within the transformer.

Reply:
If the type is coming from data-schema, do we need to refactor the definition so it's common to both data-schema and the transformer?

Reply:
This currently isn't being used. I have a WIP change that removes all supporting type definitions from data-schema. For that change, this type is necessary. Happy to remove it from this PR if you'd like; in hindsight it shouldn't have been included here.